lib/categories/FindTheThing/findthething1@Denish-Ranpariya/findthething1@HomePage.dart

```diff
@@ -0,0 +1,170 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:riddleworld/categories/FindTheThing/findthething1@Denish-Ranpariya/color_class.dart';
+import 'package:riddleworld/universal/result.dart';
+import 'package:riddleworld/universal/riddleAppbar.dart';
+
+class FindTheThing1HomeScreen extends StatefulWidget {
+  @override
+  _FindTheThing1HomeScreenState createState() =>
+      _FindTheThing1HomeScreenState();
+}
+
+class _FindTheThing1HomeScreenState extends State<FindTheThing1HomeScreen> {
+  int index = Random().nextInt(4);
+  String answer;
+  bool _isScore = false;
+  int totalScore = 0;
+  TextEditingController controller = new TextEditingController();
+
+  void resetHandler() {
+    setState(() {
+      totalScore = 0;
+      _isScore = false;
```

Comment:
In resetHandler, change the index value by assigning the Ransom.nextInt(n) function.

I mean add, 
index =  Ransom.nextInt(n); in setState of resetHandler.